cloud/app/routes/brains.py

```
@@ -0,0 +1,42 @@
+    manifest: dict = {}
+
+
+class ConnectResponse(BaseModel):
+    brain_id: str
+    status: str = "connected"
+
+
+@router.post("/brains/connect", response_model=ConnectResponse)
+async def connect_brain(
+    body: ConnectRequest,
+    brain: dict = Depends(get_current_brain),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)